src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply";
+    private static final String BUY_WORD = "buy";
+    private static final String RESULT_WORD = "result";
+    private static final int NAME_INDEX = 0;
+    private static final int NUMBER_INDEX = 1;
+    private static final String GET_SUM_SUPPLY = "getSupply";
+    private static final String GET_SUM_BUY = "getBuy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(fromFileName, toFileName);
+    }
+
+    private String[] readFile(String fileName) {
+        File file = new File(fileName);
+        String[] stringFileArr;
+        try {
+            stringFileArr = Files.readAllLines(file.toPath()).toArray(new String[0]);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + file, e);
+        }
+        return stringFileArr;
+    }
+
+    private int generateReport(String getSum, String fileName) {
+        String[] stringFileArr = readFile(fileName);
```

Comment:
not informative naming(stringFileArr, stringValue, builder, builderArr, value), how to guess what it stores?